From 9f5876fd571d2924d3ac4f1608f8826af66774ef Mon Sep 17 00:00:00 2001
From: Siva Rajesh <siva.rajesh.jarugula@xilinx.com>
Date: Thu, 17 Aug 2017 13:03:02 +0530
Subject: [PATCH 329/532] dt: bindings: drm: xilinx: dsi: Correcting HACT
 calculations by removing the pixels-per-beat property

commit 60d3991ec1a2cba9ebbdefbbb4f6ad37dd7af9a1 from
https://github.com/Xilinx/linux-xlnx.git

HACT calculaion was incorrectly using the pixels-per-beat IP parameter. Since this is no longer used, hence removed.

Signed-off-by: Siva Rajesh <siva.rajesh.jarugula@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 Documentation/devicetree/bindings/drm/xilinx/dsi.txt | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/Documentation/devicetree/bindings/drm/xilinx/dsi.txt b/Documentation/devicetree/bindings/drm/xilinx/dsi.txt
index 2e456fd..f56db6a 100644
--- a/Documentation/devicetree/bindings/drm/xilinx/dsi.txt
+++ b/Documentation/devicetree/bindings/drm/xilinx/dsi.txt
@@ -5,8 +5,6 @@ The IP core supports transmission of video data in MIPI DSI protocol.
 Required properties:
  - compatible: Should be "xlnx,mipi-dsi-tx-subsystem".
  - reg: Base address and size of the IP core.
- - xlnx,dsi-pixels-perbeat: Number of pixels per beat. The value should
-   be one of 1, 2, or 4.
  - xlnx,dsi-datatype: Color format. The value should be one of "MIPI_DSI_FMT_RGB888",
    "MIPI_DSI_FMT_RGB666", "MIPI_DSI_FMT_RGB666_PACKED" or "MIPI_DSI_FMT_RGB565".
  - simple_panel: The subnode for connected panel. This represents the
@@ -43,7 +41,6 @@ Example:
 		compatible = "xlnx,mipi-dsi-tx-subsystem";
 		reg = <0x0 0x80000000 0x0 0x10000>;
 		xlnx,dsi-num-lanes = <4>;
-		xlnx,dsi-pixels-perbeat = <1>;
 		xlnx,dsi-data-type = <MIPI_DSI_FMT_RGB888>;
 		#address-cells = <1>;
 		#size-cells = <0>;
-- 
2.7.4

